cves/CVE-2015-5963.yml

```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Someone can repeatedly make logout requests, which would cause many new empty
+  sessions to be created. This can result in a denial of service by filling up
+  the session store, and it can cause other users' session records to be evicted.
```

Comment:
Denial of service could be considered a technical term, maybe go for "interrupt service availability"